modules/saambaaBidAdapter.js

```diff
@@ -0,0 +1,414 @@
+import * as utils from '../src/utils.js';

```

Comment:
We have changed all adapters so now they are only importing the functions they use. Can you replace this import with just the ones you are using like `isFn` and `deepAccess`?